pipelines/pulpcore/02-install.yaml

```diff
@@ -0,0 +1,13 @@
+---
+- name: install pulpcore
+  hosts:
+   - "{{ forklift_server_name }}"
+  become: True
+  vars_files:
+    - ../vars/install_base.yml
+    - ../vars/forklift_{{ pipeline_type }}.yml
+  roles:
+    - epel_repositories
+    - pulp.pulp_installer.pulp_all_services
+  environment:
```

Comment:
Tis a nit, but I'd tend to expect seeing this above roles.

Reply:
fair, will update